.github/workflows/merge-pr.yml

```diff
@@ -99,14 +78,14 @@ jobs:
           sudo apt-get update
           sudo apt install -y protobuf-compiler
       - name: Install Rust Toolchain
-        uses: actions-rs/toolchain@v1
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
         with:
           default: true
           profile: minimal
           target: wasm32-unknown-unknown
           toolchain: stable
       - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@v0.1
+        uses: actions-rs/tarpaulin@044a1e5bdace8dd2f727b1af63c1d9a1d3572068
```

Comment:
Looking into a replacement for this... I used the first suggestion, but there are other coverage tools that might work better now.